examples/step-49/doc/intro.dox

```diff
@@ -99,13 +99,337 @@ many cases it is best to combine both approaches. The file can be easily
+Point(3) = {3.5, 3.36, 0, 1.0};
+Point(4) = {-3.5, 3.36, 0, 1.0};
+Point(5) = {-3.5, 0.2, 0, 1.0};
+Point(6) = {-0.98 - b, 0.2, 0, 1.0};
+Point(7) = {-0.98, 0, 0, 1.0};
+Point(8) = {-0.98 - b, -0.2, 0, 1.0};
+Point(9) = {-3.5, -0.2, 0, 1.0};
+Point(14) = {3.5, 0, 0, 1.0};
+@endcode
+
+The 2 points that separate the left and right side of the mesh
+
+@code
+Point(36) = {-0.98, 3.36, 0, 1.0};
+Point(37) = {-0.98, -3.36, 0, 1.0};
+@endcode
+
+Points for the right hand side slit
+
+@code
+Point(107) = {3.5, -1.88, 0, 1.0};
+Point(108) = {3.5, -1.48, 0, 1.0};
+Point(109) = {1.2, -1.88, 0, 1.0};
+Point(110) = {1.8, -1.48, 0, 1.0};
+@endcode
+
+Points for the triangle on the right hand side
+
+@code
+Point(111) = {3.5, 1.18, 0, 1.0};
+Point(112) = {3.5, 2.18, 0, 1.0};
+Point(113) = {3, 1.68, 0, 1.0};
+@endcode
+
+<h5>Lines on outer domain</h5>
+
+To creat lines on a mesh, you go to <code>Geometry -> Elementary
+entities -> Add -> Line</code>.  You do not get a prompt to enter in
+specific coordinates, rather you simply click a beginning point and
+ending point and it will create the line instantly.
+
+Below are the lines for the left hand side of the domain
+
+@code
+Line(4) = {4, 5};
+Line(5) = {5, 6};
+Line(6) = {6, 7};
+Line(7) = {7, 8};
+Line(8) = {8, 9};
+Line(9) = {9, 1};
+Line(14) = {7, 14};
+@endcode
+
+Lines for the bottom/top domain
+
+@code
+Line(38) = {1, 37};
+Line(39) = {37, 2};
+Line(40) = {3, 36};
+Line(41) = {36, 4};
+@endcode
+
+Lines that bisect the domain from left/right
+
+@code
+Line(42) = {37, 7};
+Line(43) = {7, 36};
+@endcode
+
+Lines for the right hand side of the domain
+
+@code
+Line(52) = {2, 107};
+Line(53) = {107, 109};
+Line(54) = {109, 110};
+Line(55) = {110, 108};
+Line(56) = {108, 14};
+Line(57) = {14, 111};
+Line(58) = {112, 3};
+Line(59) = {111, 113};
+Line(60) = {113, 112};
+@endcode
+
+<h5>Creating holes in domain</h5>
+
+Here are the points and lines that create the two holes in the domain.
+
+Relevant points for bottom left hole.
+
+@code
+Point(31) = {-2.1, -1.54, 0, 1.0}; // origin
+Point(32) = {-2.1, -0.84, 0, 1.0}; // top point
+Point(33) = {-2.1, -2.24, 0, 1.0}; // bottom point
+Point(103) = {-1.4, -1.54, 0, 1.0}; // right point
+Point(104) = {-2.8, -1.54, 0, 1.0}; // left point
+@endcode
+
+Here we use the <code>Circle arc</code> entity, which is found in the same category as
+<code>Point<\code> and <code>Lines</code>.  The file reads "Circle"
+but the difference to note is that this function uses three points to
+make an arc of a circle, namely the starting point, center, and ending point.
+Splitting the circle up like this allows for more freedom when declaring the
+boundary IDs.
+
+@code
+Circle(48) = {103, 31, 32};
+Circle(49) = {32, 31, 104};
+Circle(50) = {104, 31, 33};
+Circle(51) = {33, 31, 103};
+@endcode
+
+Below we use the <code>Circle</code> entity, which requires the origin, radius,
+and starting and ending angle respectively.
+
+@code
+rad1 = .5;
+Point(19) = {0.42, 2.0, 0, 1.0}; // origin
```

Comment:
I am confused, do these points then matter at all? They are not used? If not, please remove.